nestkernel/source_table_position.h

```diff
@@ -97,15 +106,17 @@ SourceTablePosition::wrap_position( const std::vector< std::vector< BlockVector<
       continue;
     }
 
+    // if we can not find a valid lcid by decreasing synapse or thread
+    // indices, we have read all entries
     assert( tid < 0 );
     assert( syn_id < 0 );
     assert( lcid < 0 );
-    return;
+    return; // reached the end
```

Comment:
I would comment here `reached the end without finding a valid lcid` and then have a (if unnecessary) `return` outside the while loop with a comment `found a valid lcid`

Reply:
ok